app/src/main/java/org/sopt/sample/GalleryFragment.kt

```diff
@@ -22,6 +39,50 @@ class GalleryFragment : Fragment() {
         return binding.root
     }
 
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        initRecyclerView()
+        initUserInfo()
+    }
+
+    private fun initUserInfo() {
+        userListService.getUserList().enqueue(object : Callback<ResponseUserList> {
+            override fun onResponse(
+                call: Call<ResponseUserList>,
+                response: Response<ResponseUserList>
+            ) {
+                if (response.isSuccessful) {
+                    response.body()?.let {
+                        galleryList.clear()
+                        for (user in it.data) {
+                            galleryList.add(user)
+                            Log.i("userinfo", user.toString())
+                        }
+                        Log.i("body", it.toString())
+
+                        galleryAdapter.setItems(galleryList)
```

Comment:
```suggestion
                        galleryAdapter.setItems(it.data.toList())
```
1. 우선 해당 로직에서 clear 이후 add만 하기 때문에 `galleryList`를 private var 로 설정할 이유가 없어 보입니다.
2. collection 내장 함수인 `toList` 를 사용하면 위 로직을 한줄로 줄일 수 있습니다.
    `toList` 내장 함수는 새 주소의 리스트를 반환하는 함수입니다.
3. 서버통신을 통해서 데이터를 받아서 항상 다른 주소의 리스트가 들어오기 때문에 `it.data.toList()` 를 안하고 `it.data` 만 넣어도 될 것 같다는 생각이 듭니다..!

Reply:
어차피 들어올때마다 바뀌니까 galleryList가 필요없는 거였군용 galleryList 변수 삭제하고 로직 수정하겠습니당~~